tests/functions/test_is_connected.py

```diff
@@ -0,0 +1,8 @@
+from janitor.functions import is_connected
```

Comment:
If you move the `is_connected` function to `utils.py`, could you also move this corresponding test too?

Reply:
**Action item:** As per comment above.